<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>matrixFromSequence</class>
 <widget class="QDialog" name="matrixFromSequence">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Matrix From Sequence</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <widget class="QSpinBox" name="intLength">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QCheckBox" name="checkBox">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Do you know the length 
 of your sequence?</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Length:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Start Position:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="intPosition">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Save Matrix as:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="chooseSequenceOrigin">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Not implemented in Qt</string>
       </property>
       <item>
        <property name="text">
         <string>You have the binding sequences seperately in a fasta file.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>You have a file with genomic coordinates of the sequences.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>You want to analyze the result of a previous analysis.</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;How would you like to obtain &lt;/p&gt;&lt;p&gt;the binding sequences to analyze?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="chooseSourceSequence">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Not implemented in Qt</string>
       </property>
       <item>
        <property name="text">
         <string>You have the binding sequences as a list in a file.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>You want to analyze the result of a previous analysis.</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="chooseMatrix">
       <item>
        <property name="text">
         <string>Standard Probability Matrix</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Normalized Probability Matrix</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Standard Logarithmic Matrix</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Normalized Logarithmic Matrix</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>How would you like to obtain the binding 
 sequences to analyze?</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="buttonSave">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
